source/conf-mysql-requirements.rst

```diff
@@ -110,10 +105,14 @@ follows:
    GRANT SELECT, PROCESS, SUPER, REPLICATION CLIENT, RELOAD ON *.* TO 'pmm'@' localhost' IDENTIFIED BY 'pass' WITH MAX_USER_CONNECTIONS 10;
    GRANT SELECT, UPDATE, DELETE, DROP ON performance_schema.* TO 'pmm'@'localhost';
 
+.. seealso::
+      :ref:`pmm-admin.add-mysql-metrics` - Using the |pmm-admin.add| command
+      to add a monitoring service
+
 
-For more information, run as root
+For more information, run:
 |pmm-admin.add|
 |opt.mysql|
-|opt.help|.
+|opt.help|
```

Comment:
There's still something wrong here...on the website this does not render properly but in my local env it does...created https://jira.percona.com/browse/PMM-5288 to track.